build.gradle

```diff
@@ -396,7 +404,11 @@ project(':ambry-rest') {
         compile "io.dropwizard.metrics:metrics-jmx:$metricsVersion"
         compile "net.sf.jopt-simple:jopt-simple:$joptSimpleVersion"
         compile "io.netty:netty-all:$nettyVersion"
-        compile "io.netty:netty-tcnative-boringssl-static:$nettyTcnativeVersion"
+        runtime "io.netty:netty-tcnative-boringssl-static:$nettyTcnativeVersion:linux-x86_64"
```

Comment:
what's the difference between runtime and runtimeOnly?

Reply:
Not very familiar with Gradle, but it seems like runtime is a deprecated configuration, just like compile. The new gradle would use implementation to replace compile and runtimeOnly to replace runtime.

Reply:
https://docs.gradle.org/6.8.1/userguide/java_plugin.html#tab:configurations